From 1b3daff0c0a2134fbb32ecac1ea923a637d6674c Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:06:02 -0400
Subject: [tools] perf/top: Fix documentation of invalid -s option

Message-id: <1410181752-25631-118-git-send-email-jolsa@redhat.com>
Patchwork-id: 91770
O-Subject: [PATCH RHEL7.1 BZ1134356 117/307] perf top: Fix documentation of invalid -s option
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 5e09714b0e877d5d062530aae4061074d864243f
Author: Namhyung Kim <namhyung@kernel.org>
Date: Wed Mar 19 10:46:18 2014 +0900

description
===========
On perf top, the -s option is used for --sort, but the man page
contains invalid documentation of -s option for --sym-annotate.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-top.txt b/tools/perf/Documentation/perf-top.txt
index cdd8d49..976b00c 100644
--- a/tools/perf/Documentation/perf-top.txt
+++ b/tools/perf/Documentation/perf-top.txt
@@ -87,7 +87,6 @@ Default is to monitor all CPUS.
 --realtime=<priority>::
  Collect data with this RT SCHED_FIFO priority.
 
--s <symbol>::
 --sym-annotate=<symbol>::
         Annotate this symbol.
 
-- 
1.7.1